Mathlib/CategoryTheory/Pi/MonoidalClosed.lean

```diff
@@ -0,0 +1,59 @@
+/-
+Copyright (c) 2025 Sina Hazratpour. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sina Hazratpour
+-/
+
+import Mathlib.CategoryTheory.Closed.Monoidal
+import Mathlib.CategoryTheory.Pi.Monoidal
+
+
```

Comment:
No blank line after the copyright header, exactly one blank line before the module doc.
```suggestion
```